### ISSUE TITLE
Installation for all users?

### ISSUE THREAD

**Issue by CRCinAU** [chris-marsh/pureline#23]
Is there any hints / howto on how to install this for all users?

I would get that the source should exist in /etc/profile.d/something

Then install under /usr/local? or /opt?

**Comment by chris-marsh** [comment]
Yep your right, PureLine should have a proper installation method. I'm going to try and commit a bit more time to the project and aim for a feature freeze and stable release. Then it should easier to work on distro packages to install system wide.

**Comment by max06** [comment]
I've just tried this - with not very much success.

```bash
    # source external segments
    local segment_dir
    segment_dir=$(dirname "${BASH_SOURCE[0]}")'/segments'
```

Sourcing `/opt/pureline/pureline` fails to load the segments, since `${BASH_SOURCE[0]}` points to `/etc/profile.d/pureline.sh`.

Is there a better way? 

(Instead of sourcing 2 files, I just source pureline in the profile script and directly add the configuration content afterwards.)

**Comment by max06** [comment]
Nevermind... Adding my config into the profile-script instead of sourcing it was my mistake.

Out of interest... can someone explain me the difference between 

```bash
source a.sh b.sh
```

and 

```bash
source a.sh
source b.sh
```

? 😅
